src/tracks/playabletrack/wavetrack/ui/GetWaveDisplay.cpp

```diff
@@ -196,11 +199,11 @@ bool GetWaveDisplay(const Sequence &sequence,
          if (diff > 0) {
             MinMaxSumsq values(temp.get(), diff, divisor);
             const int lastPixel = pixel - 1;
-            float &lastMin = min[lastPixel];
+            float &lastMin = columns[lastPixel].min;
             lastMin = std::min(lastMin, values.min);
-            float &lastMax = max[lastPixel];
+            float &lastMax = columns[lastPixel].max;
             lastMax = std::max(lastMax, values.max);
-            float &lastRms = rms[lastPixel];
+            float &lastRms = columns[lastPixel].rms;
```

Comment:
I would precede all this with `const auto &column = columns[lastPixel];` just to eliminate the need for me to read the repeated subscripting.  But maybe compilers are smart enough to eliminate the implied repeated code.  (Calling Mr. Godbolt?)